hugegraph-dist/src/assembly/static/bin/hugegraph

```diff
@@ -37,10 +37,19 @@
 ### END INIT INFO
 
 # Variables
-INSTALL_DIR=
```

Comment:
change the comment, it requires user to set a **fixed abs path** manually (should't use a relative path)

Reply:
You are right. and I think should add check for `Variables`